### PR TITLE
Help the compiler unify

### PR DESCRIPTION
**Pull request by kevinresol**
In 3.3, FunctionArg has a new member `meta`

### CODE REVIEW

**Comment by waneck** [comment]
Do you have any idea why this failed on travis? https://travis-ci.org/waneck/mcli/jobs/132795713

**Comment by kevinresol** [comment]
I have no idea. It builds fine on my mac machine, lastest haxe

File: mcli/internal/Macro.hx [hunk 1]
```diff
@@ -211,7 +211,7 @@ class Macro
 					} else if (last != null) {
 						args.push(last);
 					}
-					Function(args.map(function(a) return { name: a.name, opt: a.opt, t: convert(a.t, f.pos) }), varArg);
+					Function(args.map(function(a):FunctionArg return { name: a.name, opt: a.opt, t: convert(a.t, f.pos) }), varArg);
```
Comment (ciscoheat) [inline mcli/internal/Macro.hx]: This didn't fix the 3.3 problem for me, I had to add `meta: null` to line 245.

Comment (waneck) [inline mcli/internal/Macro.hx]: can you add a PR for that?

Comment (ciscoheat) [inline mcli/internal/Macro.hx]: Sure, I'll do that tomorrow!

Comment (kevinresol) [inline mcli/internal/Macro.hx]: That is really strange, this PR worked for me. :confused:

**Comment by ciscoheat** [comment]
@kevinresol does my #10 work for you as well (without your fix applied)?

**Comment by kevinresol** [comment]
!!!!!!!!

My bad. This PR is totally wrong. I meant to add the type hint at line 245. I think I made this PR change on Github's editor where I made the mistake. I have been using [my own fork](https://github.com/kevinresol/mcli/commit/e8d7e40f918e7ae9b43535c498929816ca9880f9#diff-8a1d78635ae1d24bd4da04bafaef601bR245) for some time so I didn't notice this has been wrong from the very beginning. A big apologize for that.

**Pull request closed without merging**